### ISSUE TITLE
get cellbender version from command line

### ISSUE THREAD

**Issue by edgano** [broadinstitute/CellBender#157]
Hi guys, great job!
I was wondering if there is a flag `-v/--version` to capture cellbender version from the command line.
We will like to document all software versions for the report of the analysis.

Thanks!

**Comment by sjfleming** [comment]
Thanks @edgano !  There is not yet such a flag in the current releases.  But that is going to be part of the next v0.3.0 release.  I agree with you that it's important.  (Currently v0.3.0 is being worked on in the `sf_dev_0.3.0_postreg` branch)

The behavior is like this

```console
$ cellbender -v
```
or
```console
$ cellbender --version
```
gives the version number, like `0.3.0`, and then also whenever you run the `remove-background` tool, you get 
```console
$ cellbender remove-background --input test.h5 --output test_out.h5

cellbender:remove-background: Command:
cellbender remove-background --input /Users/sfleming/Desktop/test.h5 --output /Users/sfleming/Desktop/test_out.h5
cellbender:remove-background: CellBender 0.3.0
cellbender:remove-background: (Workflow hash 940b0aac8e)
cellbender:remove-background: 2022-09-23 08:57:02
cellbender:remove-background: Running remove-background
cellbender:remove-background: Loading data from /Users/sfleming/Desktop/test.h5
cellbender:remove-background: CellRanger v3 format
...
```

**Comment by sjfleming** [comment]
Closed by #238
